### PR TITLE
bloop: 1.1.1 -> 1.1.2

### PR DESCRIPTION
**Pull request by Tomahna**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/tools/build-managers/bloop/default.nix [hunk 1]
```diff
@@ -16,12 +16,12 @@ let
...
     outputHashMode = "recursive";
     outputHashAlgo = "sha256";
-    outputHash     = "0w2gk9ladkzm3mx3qsmjqjakj94g7s4s3xfzl6yny1h4dg1456n6";
+    outputHash     = "0ki4xh8z31nv42jik998r7gvsxcm0vysx83w0nyvbhi18cah1qh6";
   };
 in
 stdenv.mkDerivation rec {
   name = "${baseName}-${version}";
-  nailgunCommit = "6992a3bf";
+  nailgunCommit = "dc1dd806";
```
Comment (alyssais) [inline pkgs/development/tools/build-managers/bloop/default.nix]: I’m curious — where does this hash come from?

Comment (Tomahna) [reply pkgs/development/tools/build-managers/bloop/default.nix]: From the official installation script https://github.com/scalacenter/bloop/releases/download/v1.1.2/install.py

Comment (alyssais) [reply pkgs/development/tools/build-managers/bloop/default.nix]: Ah, cool. Would you mind adding a comment saying that, so somebody who doesn’t know the package would be able to update it more easily?

Comment (alyssais) [reply pkgs/development/tools/build-managers/bloop/default.nix]: Ah, cool. Would you mind adding a comment saying that, so somebody who doesn’t know the package would be able to update it more easily?

Comment (Tomahna) [reply pkgs/development/tools/build-managers/bloop/default.nix]: Done

Comment (alyssais) [reply pkgs/development/tools/build-managers/bloop/default.nix]: Thanks!

**Comment by alyssais** [comment]
@GrahamcOfBorg build bloop

**Pull request merged**